framework/summary_model/src/lib.rs

```diff
@@ -97,21 +97,33 @@ impl RunSummary {
         self.env.insert(key, value);
     }
 
+    /// Computer a fingerprint for this run summary
+    ///
+    /// The fingerprint is intended to uniquely identify the configuration used to run the scenario.
+    /// It uses the
+    ///     - Scenario name
+    ///     - Run duration
+    ///     - Assigned behaviours
+    ///     - Selected environment variables
+    ///     - Wind Tunnel version
+    ///
+    /// The fingerprint is computed using SHA3-256.
```

Comment:
I wonder if a link to the type makes sense here?

```suggestion
    /// The fingerprint is computed using [sha3::Sha3_256].
```